app/models/account.rb

```diff
@@ -1,3 +1,9 @@
 class Account < ApplicationRecord
   belongs_to :project
+  belongs_to :account_manager, class_name: 'User'
+
+  validates :name, presence: true
+  validates :services, presence: true, allow_nil: false
+  validates :account_manager_id, allow_nil: false
+  validates :owner_request, presence: true, allow_nil: false
```

Comment:
Szerintem ebben a formában ez a validáció csak IGAZ boolean értékeket fogad el.
```ruby
false.present?
=> false
```

Boolean változó jelenlétét így tudod ellenőrizni:
```ruby
validates :owner_request, inclusion: { in: [true, false] }
```

Reply:
Csak nil-t nem engedne, mert ha nem jelölünk be rajta semmit akkor false lesz az érték, de ha random generálunk a DB-be valamit vagy bele túrunk akkor így nem fogja engedni a nil-t.
A tied amúgy szebb, arra cserélem.